lib/openstudio-standards/standards/Standards.SpaceType.rb

```diff
@@ -575,6 +575,63 @@ def space_type_apply_int_loads_prm(space_type, model)
     end
   end
 
+  # Modify the lighting schedules for Appendix G PRM for 2016 and later
+  #def space_type_light_sch_change(space_type, model)
+  #  space_type.lights.sort.each do |lgts|
+  #    lgts.schedule.get.to_ScheduleRuleset.get.scheduleRules.each do |week_rule|
+  #      day_rule = week_rule.daySchedule()
+  #      # clone the current lighting schedule to new schedule
+  #      new_day_rule = day_rule.clone(model).to_ScheduleDay.get
+  #      #day_rule = week_rule.daySchedule()
+  #      times = new_day_rule.times()
+  #      times.each do |time|
+  #        old_value = new_day_rule.getValue(time)
+  #        new_day_rule.removeValue(time)
+  #        new_value = old_value / (1.0 - space_type_properties['occup_sensor_savings'].to_f)
+  #        new_day_rule.addValue(time,new_value)
+  #      end
+  #      
+  #    end
+  #  end
+  #end
+
+  # Modify the lighting schedules for Appendix G PRM for 2016 and later
```

Comment:
Could you please add appropriate YARN comments?

Reply:
Modified.